extensions/amp-analytics/0.1/variables.js

```diff
@@ -294,7 +294,7 @@ export class VariableService {
     }
 
     if (isArray(raw)) {
-      return raw.map(encodeURIComponent).join(',');
+      return raw.map(this.encodeVars.bind(this)).join(',');
```

Comment:
the type checker doesn't seem to like this. ` ERROR - actual parameter 1 of raw.map does not match formal parameter`

Maybe pull out into separate variable??